src/components/Avatar/Avatar.story.jsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, number } from '@storybook/addon-knobs';
+import { generateVariantsFor } from '../../utils/propsVariants';
+import Avatar from './Avatar';
+
+const stories = storiesOf('Avatar', module);
+
+stories.add(
+  'Avatar',
+  () => {
+    const variants = generateVariantsFor({
+      size: [null, 19, 20, 200, 201],
+      userName: [null, 'Jacob N. Ross', ''],
+      userEmail: [null, 'xuzhu@d3b.center', ''],
+      imgUrl: [null, 'https://www.w3schools.com/css/img_avatar.png', ''],
+    });
+    return (
+      <section>
+        <Avatar
+          className="hide-in-tests"
+          userName={text('User Name', 'Jacob N. Ross')}
+          userEmail={text('User Email', 'smilemystellar@gmail.com')}
+          imgUrl={text('Profile Image URL', 'https://www.w3schools.com/css/img_avatar.png')}
+          size={number('Avatar Size', 40)}
+        />
+        <div className="show-in-tests hidden">
```

Comment:
so all of these were showing up in percy but just getting cutoff due to the snapshot size. 

add `style={{width: 1280}}`, this is the max-width of percy snapshots